From f55948c353961689d95ca3cd15ef9dd754bae3f1 Mon Sep 17 00:00:00 2001
From: Guillaume Papin <guillaume.papin@epitech.eu>
Date: Thu, 10 May 2018 22:29:47 +0200
Subject: [PATCH 2/2] remove Git revision checks

The system git was found if installed on the system
(means potentially different behavior on different builders).

And the sources are downloaded from an archive, patched,
so there is no point in this git revision.
---
 CMakeLists.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3debf77..4b33d86 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,16 +21,6 @@ else()
   message(STATUS "IWYU in-tree configuration")
 endif()
 
-# Pick up Git revision so we can report it in version information.
-find_package(Git)
-if( GIT_FOUND AND EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.git" )
-  execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --short HEAD
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    OUTPUT_VARIABLE IWYU_GIT_REV
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-  message(STATUS "IWYU git version: ${IWYU_GIT_REV}")
-endif()
-
 add_llvm_executable(include-what-you-use
   iwyu.cc
   iwyu_ast_util.cc
-- 
2.10.0

